num2words/lang_PT.py

```diff
@@ -21,18 +21,36 @@
 
 from .lang_EU import Num2Word_EU
 
-DOLLAR = ('dólar', 'dólares')
-CENTS = ('cêntimo', 'cêntimos')
+GENERIC_DOLLARS = ('dólar', 'dólares')
+GENERIC_CENTS = ('cêntimo', 'cêntimos')
 
 
 class Num2Word_PT(Num2Word_EU):
 
     CURRENCY_FORMS = {
-        'AUD': (DOLLAR, CENTS),
-        'CAD': (DOLLAR, CENTS),
-        'EUR': (('euro', 'euros'), CENTS),
+        'AUD': (GENERIC_DOLLARS, GENERIC_CENTS),
+        'CAD': (GENERIC_DOLLARS, GENERIC_CENTS),
+        'EUR': (('euro', 'euros'), GENERIC_CENTS),
         'GBP': (('libra', 'libras'), ('péni', 'pence')),
-        'USD': (DOLLAR, CENTS),
+        'USD': (GENERIC_DOLLARS, GENERIC_CENTS),
+        'RUB': (('rublo', 'rublos')), GENERIC_CENTS),
```

Comment:
This extra bracket is causing a SyntaxError. See the test logs: https://travis-ci.org/github/savoirfairelinux/num2words/jobs/759299965.
```suggestion
        'RUB': (('rublo', 'rublos'), GENERIC_CENTS),
```

Reply:
👍🏼